examples/js/exporters/GLTFExporter.js

```diff
@@ -1648,7 +1648,7 @@ THREE.GLTFExporter.prototype = {
 			var skeleton = object.skeleton;
 			var rootJoint = object.skeleton.bones[ 0 ];
```

Comment:
Wouldn't this line wail if `skeleton` was `undefined`?

Reply:
I get  TypeError: Cannot read property 'bones' of undefined
    at processSkin (GLTFExporter.js:1649)
    at processInput (GLTFExporter.js:1993)
    at ./node_modules/three/examples/js/exporters/GLTFExporter.js.THREE.GLTFExporter.parse (GLTFExporter.js:2005)
    at _callee2$ (ModelBuilder.js:255)
    at tryCatch (runtime.js:45)
    at Generator.invoke [as _invoke] (runtime.js:271)
    at Generator.prototype.<computed> [as next] (runtime.js:97)
    at tryCatch (runtime.js:45)
    at invoke (runtime.js:135)
    at runtime.js:145

Reply:
The patch needs to be implemented differently. Maybe like so:
```js
var skeleton = object.skeleton;

if ( skeleton === undefined ) return null;

var rootJoint = skeleton.bones[ 0 ];

if ( rootJoint === undefined ) return null;
```

Reply:
agreed.  submitting new PR